tools-public/toolpad/components/PieChart.tsx

```diff
@@ -53,5 +58,9 @@ export default createComponent(PieChartExport, {
         { name: 'Group D', value: 200 },
       ],
     },
+    loading: {
```

Comment:
Should add

```tsx
  loadingPropSource: ['data'],
  loadingProp: 'loading',
```

to the component config. Then you won't have to hook up 

```yaml
        loading:
          $$jsExpression: downloadsVersions.isLoading || false
```

Reply:
Oh, I didn't know, it's indeed documented in https://mui.com/toolpad/reference/api/create-component/#componentconfig